src/main/java/graphql/analysis/QueryTransformer.java

```diff
@@ -1,5 +1,6 @@
 package graphql.analysis;
```

Comment:
Update file to bring in line with v21 - it's because ConditionalNodes are referenced in the ExecutableNormalizedOperationFactory, and those changes are viral in a few other files